<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SoVolumeRender_ctrl</class>
 <widget class="QDialog" name="SoVolumeRender_ctrl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>235</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SoVolumeRender control</string>
  </property>
  <widget class="QLabel" name="TextLabel6">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>83</y>
     <width>120</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>interpolation</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="TextLabel6_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>121</y>
     <width>120</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>composition</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="compositionCombo">
   <property name="geometry">
    <rect>
     <x>196</x>
     <y>120</y>
     <width>292</width>
     <height>27</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>MAX_INTENSITY</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>SUM_INTENSITY</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ALPHA_BLENDING</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="interpolationCombo">
   <property name="geometry">
    <rect>
     <x>196</x>
     <y>84</y>
     <width>292</width>
     <height>27</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>NEAREST</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>LINEAR</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="TextLabel5">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>50</y>
     <width>136</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>numSlicesControl</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="numSlicesEdit">
   <property name="geometry">
    <rect>
     <x>502</x>
     <y>18</y>
     <width>52</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QSlider" name="numSlicesSlider">
   <property name="geometry">
    <rect>
     <x>192</x>
     <y>19</y>
     <width>295</width>
     <height>24</height>
    </rect>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="numSlicesControlCombo">
   <property name="geometry">
    <rect>
     <x>196</x>
     <y>50</y>
     <width>292</width>
     <height>27</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>ALL</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>MANUAL</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>AUTOMATIC</string>
    </property>
   </item>
  </widget>
  <widget class="QCheckBox" name="viewAlignedSlicesCheckBox">
   <property name="geometry">
    <rect>
     <x>73</x>
     <y>183</y>
     <width>196</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>viewAlignedSlices</string>
   </property>
  </widget>
  <widget class="QLabel" name="numSlicesLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>17</y>
     <width>120</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>numSlices</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <resources/>
 <connections/>
</ui>
